src/assets/css/twitterwall.css

```diff
@@ -24,6 +24,7 @@
 	
 }
 .image{
+	box-sizing: border-box;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The box-sizing property isn't supported in IE6 and IE7.](https://www.codacy.com/app/triplez/open-event-webapp/file/2416367476/issues/source?bid=3233857&fileBranchId=3235518#l27)